packages/nextjs/pages/_app.tsx

```diff
@@ -58,9 +58,9 @@ const ScaffoldEthApp = ({ Component, pageProps }: AppProps) => {
               <Header />
               <main className="relative flex flex-col flex-1 p-8">
                 <p>
-                Please on-board by following this{" "}
+                  Please on-board by following this{" "}
                   <a
-                    href="https://www.notion.so/tinglik/On-boarding-7-Proxies-Network-0967d277dfee49dfb2e537a1b89961b6?pvs=4"
+                    href="https://tinglik.notion.site/On-boarding-7-Proxies-Network-0967d277dfee49dfb2e537a1b89961b6"
```

Comment:
Just a remark, nothing wrong with the code. 

Once we will find the final name of the product we should move to a project page instead of your personal one. For now, it's ok, it's clean for demoing.